addons/mail/data/ir_config_parameter_data.xml

```diff
@@ -13,4 +13,10 @@
         <field name="value">bounce</field>
     </record>
 
+    <!-- Notifications -->
+    <record id="icp_mail_default_from" model="ir.config_parameter">
```

Comment:
Is there any reason not to define it directly in base, as it is used in ir_mail_server ?

Reply:
I put it there because `mail.bounce.alias` is also defined in mail, but is used in base

Reply:
Good idea :)

Reply:
So, define them the same way of the other default ICP. No need to define them as data.
https://github.com/odoo/odoo/blob/c47b0054b8824c4a034b6ce36703e20295a411d2/odoo/addons/base/models/ir_config_parameter.py#L18-L25

Reply:
Would say no and use real data, like all odoo data (allowing to have an xml_id, update, ...).